platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -6,7 +6,7 @@ dependencies {
     compile rootProject.ext.dep.supportDesign
     compile rootProject.ext.dep.timber
     compile rootProject.ext.dep.okhttp3
-    compile(rootProject.ext.dep.lost) {
+    provided(rootProject.ext.dep.lost) {
```

Comment:
@zugaldia Nope, mainly because [`LocationSource`](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationSource.java) is part of the `public` API. We should deprecate `LocationSource` and eventually remove it, at that point we'll be able to completely remove that dependency and rely solely on MAS.